website/content/docs/pipelines/index.mdx

```diff
@@ -0,0 +1,439 @@
+Teams need to run tests after a build, security scan artifacts prior to a deploy, and
+so on. Custom Pipelines enable teams to solve these problems. With this tech preview, users can
+define pipelines to run in the order they require to deliver their
+application. Additionally, pipelines allow for custom execution steps through
+the "Exec" plugin, allowing teams to run scripts as part of the pipeline
+execution, further enabling highly-customized operations.
+
+Pipelines allow users to tell Waypoint what order these operations should
+run in and configure any operation options necessary, like disabling an
+artifact push in the `build` step, or pruning old deployments
+on a release in the `release` step.
+
+Please visit the [Pipeline Configuration Docs](/docs/pipelines/config)
+for more information about the configuration of a Pipeline for a
+Waypoint project.
+
+### Pipeline Runs
+
+Each time a pipeline is run, Waypoint keeps track of the steps and run results.
+As you can see from the [waypoint pipeline run console example output](/docs/pipelines#examples),
+Waypoint identifies how many steps are in the pipeline and how many are successfully
+completed. Users can then further deep dive into specific pipeline runs to view
+more information.
+
+`waypoint pipeline list` shows the current steps, last run start and complete
+times, last run state, and total runs for every pipeline in the project.
+
+```shell
+❯ waypoint pipeline list
+
+» Waypoint Pipelines for my-project
+              ID             | NAME |    OWNER     | CURRENT STEPS | LAST RUN STARTED | LAST RUN COMPLETED |  STATE  | TOTAL RUNS
+-----------------------------+------+--------------+---------------+------------------+--------------------+---------+-------------
+  01GBBBC63TR5DYQXQVGJPS8JGZ | test | example-java |             2 | 5 days ago       | 5 days ago         | success |          5
+```
+
+`waypoint pipeline inspect` can look up a specific pipeline by ID (default) or
+name, as well as an additional option to view a specific run of that pipeline.
+Pipeline run information will include run state, steps, timestamps, and associated job IDs.
+
+```shell
+❯ waypoint pipeline inspect -name=test
+
+
+» Pipeline Configuration
+                  ID: 01GBBBC63TR5DYQXQVGJPS8JGZ
+                Name: test
+               Owner: example-java
+      Root Step Name: build
+         Total Steps: 2
+          Total Runs: 9
+    Last Run Started: 1 minute ago
+  Last Run Completed: a long while ago
+     Last Run Status: RUNNING
+```
+
+```shell
+❯ waypoint pipeline inspect -name=test -run=3
+
+» Pipeline Configuration
+              ID: 01GBBBC63TR5DYQXQVGJPS8JGZ
+            Name: test
+           Owner: example-java
+  Root Step Name: build
+     Total Steps: 2
+    Run Sequence: 3
+     Jobs Queued: [id:"01GBBBQJZHZVMKRAQSXGYXKEZY" id:"01GBBBQJZH05YD6MN91X30G7HF"]
+     Run Started: 5 days ago
+   Run Completed: 5 days ago
```

Comment:
Do the "Run Started" and "Run Completed" times mean the start/end time of the most recent run?

Reply:
It would be the timestamp of this specific run sequence, number 3.